src/project-1.js

```diff
@@ -3,137 +3,166 @@
 const multiplyByTen = (num) => {
   // return num after multiplying it by ten
   // code here
+  return num * 10;
 };
 
 const subtractFive = (num) => {
   // return num after subtracting five
   // code here
+  return num - 5;
 };
 
 const areSameLength = (str1, str2) => {
   // return true if the two strings have the same length
   // otherwise return false
   // code here
+  if (str1.length === str2.length) {
```

Comment:
You can simplify:
```
  if (str1.length === str2.length) {
     return true;
   }
   return false;
```
to:
```
return str1.length === str2.length;
```
The expression is automatically simplified into `true` or `false`.  You could then use an arrow function to simplify it further:
```
const areSameLength = (str1, str2) => str1.length === str2.length;
```